invenio_github/api.py

```diff
@@ -73,10 +75,8 @@ def api(self):
 
     @cached_property
     def access_token(self):
-        """Return OAuth access token."""
-        if self.user_id:
-            return RemoteToken.get(self.user_id, self.remote.consumer_key).access_token
-        return self.remote.get_request_token()[0]
```

Comment:
I think we're using this token getter:

https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/handlers/utils.py#L157

It sets the `RemoteToken` in the session and returns the `RemoteToken.token()`, which is a tuple: https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/models.py#L140

Since both are returning `RemoteToken.get()`, they should be similar. However the `get_request_token()` sets it in the session, if it exists.

Nevertheless, I will revert the change :)